pkg/user/signer_test.go

```diff
@@ -43,7 +44,7 @@ func (s *SignerTestSuite) SetupSuite() {
 	s.Require().NoError(err)
 	addr, err := rec.GetAddress()
 	s.Require().NoError(err)
-	s.signer, err = user.SetupSigner(s.ctx.GoContext(), s.ctx.Keyring, s.ctx.GRPCClient, addr, s.encCfg)
+	s.signer, err = user.SetupSigner(s.ctx.GoContext(), s.ctx.Keyring, s.ctx.GRPCClient, addr, s.encCfg, user.WithGasMultiplier(1.2))
```

Comment:
[question] if this test needs a gas multiplier of 1.2 should we consider bumping the default to 1.2?

Reply:
It was added just to overwrite the default value.

Reply:
ahh got it. Okay nvm